modules/apps/emm/modules/user.js

```diff
@@ -843,13 +844,29 @@ var user = (function () {
             if(!authStatus) {
                 return null;
             }
-            var user =  this.getUser({'userid': ctx.username, login:true});
-//            var result = driver.query(sqlscripts.tenantplatformfeatures.select1,  stringify(user.tenantId));
-//            if(result[0].record_count == 0) {
-//				for(var i = 1; i < 13; i++) {
-//                    var result = driver.query(sqlscripts.tenantplatformfeatures.select2, stringify(user.tenantId), i);
-//				}
-//			}
+
+            try {
+                var domain = userCoreUtil.getDomainFromThreadLocal();
```

Comment:
This has been discussed and reviewed with both jaggery team and EMM team. The reason this is used because there was an issue occurred due to usage of secondary user store.